README.md

```diff
@@ -568,6 +568,27 @@ TopSecret.configure do |config|
 end
 ```
 
+### Caching the MITIE model
+
+By default, each `TopSecret::Text` instance creates a new MITIE model, which can be slow. To improve performance, you can cache a single shared model instance:
+
+```ruby
+TopSecret.configure do |config|
+  config.shared_model = Mitie::NER.new(config.model_path)
+end
+```
+
+This is particularly useful in Rails applications. Add this to an initializer to load the model once at boot:
+
+```ruby
+# config/initializers/top_secret.rb
+Rails.application.config.after_initialize do
+  TopSecret.shared_model = Mitie::NER.new(config.model_path) if config.model_path
```

Comment:
In theory this works, but if you're using [Trove](https://github.com/ankane/trove), and [pulling during asset pre-compilation](https://github.com/ankane/trove), the file will not exist during initialization.